transcom/mymove
# INT B-18859 Prime Simulator Display HHG Pro Gear

## [B-18859](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A907530)

## Summary

A user should now be able to update an HHG's Pro Gear weights from the Update Shipment form in the Prime Simulator. Depends on the recent changes that added pro gear weights to mtoShipment and makes them updateable via the prime API.

Downstream dependency of B-18840 and B-18669

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Ensure at least one move, containing at least one HHG shipment, exists and is accessible to the Prime User. You can probably use an existing move created by dev_db_e2e_populate.
2. Access the prime simulator and open that move.
3. Verify there are rows to display both pro gear weights near the other weights listed for that shipment.
4. Select the "Update Shipment"
5. Fill out any required fields as necessary, the date fields can't be blank to submit the form.
6. Enter values for the Pro Gear Weight and Spouse Pro Gear Weight fields.
7. Save the form.
8. Verify your new values are displayed in the location you verified existed in step 3.
9. Re-open the form to verify the values persist.
10. Open a new move that contains a PPM shipment (ex: MMOVEL)
11. Verify the rows you checked for in step 3 are not displayed since PPMs display their pro gear weights elsewhere.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
